composer.json

```diff
@@ -34,7 +34,10 @@
   "autoload": {
     "psr-4": {
       "Localheinz\\OrganizingTestCodeInPhp\\": "src/"
-    }
+    },
+    "exclude-from-classmap": [
+      "/src/Test/"
+    ]
```

Comment:
When you declare test code in subdirectories of production code, managing exclusions for the `composer` autoloader in `composer.json ` is less work.